### DISCUSSION TITLE
试用了一下 Ant Design Mobile@5，感觉非常难受

### DISCUSSION THREAD

**Discussion by tangwenru** [ant-design/ant-design-mobile#5927] [General]
### Version of antd-mobile

5.27.0

### What is this feature about?

- 1、为什么 Ant Design Mobile 很多特性，不和 Ant Design 保持一致？
  - 比如 《Button type="primary" /》 和  《Button  color="primary" /》，为何不统一？
  - 比如 《Divider direction="vertical" /》  和 《Divider type="vertical" /》 
  - 再比如 Tabs ，Form 很多属性 叫的名字一样，但是 属性值差别很大；
  - 为什么 要 给开发者 增加如此多的切换成本呢？（可能  Ant Design Mobile 本身的开发者自己随便起名字爽了，但是千千万万的  Ant Design Mobile 使用者就非常难受）
- 2、Ant Design Mobile 组件的属性 远比 Ant Design 要少很多，粗糙很多，这个估计人力投入就不够；
- 3、插件能尽可能细致一些么？
  - 我怀疑 Ant Design Mobile 的开发者很少使用Ant Design ；
  - 比如 antd-mobile 的 《Tag》标签，自定义 color属性  一律 字体 白色！ 这意味着，如果没有匹配到颜色，背景就透明了，并且字体白色，意味着 啥都看不见！   为何不计算一下 color 和 bgColor 的差值，做一个反相？  这个问题，在 antd 里面就不存在；
- 4、夜深了，就说这么多吧，总之希望 antd-mobile 能做得细致点，虽然现在比以前好很多了，但这个库一直比较挫，都搓了好多年了； 

不知道 at 谁，暂就 at 你们吧，😊
@chenkan kan @afc163

**Comment by qiaogaolong** [comment]
@tangwenru  感觉v5又要放弃了，现在已转型，感觉好多细节还没有其他组件库做的好，阿里不是当初的阿里了感觉

**Comment by zombieJ** [comment]
感谢反馈，关于 API 不一致的问题我觉得可以展开说说。在 antd 开发过程中，随着功能累加逐渐出现相同类型组件 API 不一致的情况，我们在每个大版本都会合并（比如 `visible` 和 `open`），但是同时仍然会有一些不一致的情况（例如 Select 的 `options` 和 TreeSelect 的 `treeData`）。为了避免 breaking change，一般 API 收敛会在大版本中逐个替换。而在 mobile 上，同样需要一些时间来做收敛，这是必须做的。

关于功能问题，antd 有非常活跃的社区成员共同维护。而在 mobile 侧，目前社区活跃度相对捉襟见肘。也欢迎参与共建，来成为社区的一员。

另外，使用问题欢迎开新的 issue。这个讨论我转到 Discussions 里去了。
